src/libData/AccountData/AccountStoreSC.tpp

```diff
@@ -443,6 +463,26 @@ std::string AccountStoreSC<MAP>::GetCallContractCmdStr(
   return ret;
 }
 
+template <class MAP>
+bool AccountStoreSC<MAP>::ParseContractCheckerOutput(
```

Comment:
Do we plan to do something with the checker output eventually?  It currently seems unused apart from seeing if it's parse-able here.

Reply:
The parse will only check whether the output from Scilla-Checker can form a JSON object or not. If can then it means passed, otherwise it will be a error message which can not be converted to JSON